packages/appstore/src/components/real-account-creation-banner/index.tsx

```diff
@@ -0,0 +1,4 @@
+import RealAccountCreationBanner from './real-account-creation-banner';
+import './real-account-creation-banner.scss';
```

Comment:
@mitra-deriv it's not an error, but can lead to this. I'm prefer to add css import inside of the component which uses this css file. I'll give you two examples how you can import this component in another place:

1. import ReacAccountCreationBanner from './../components/real-account-creation-banner';
2. import ReacAccountCreationBanner from './../components/real-account-creation-banner/real-account-creation-banner';

Component will be imported correct in 2 ways, but with your approach 2nd import will not download css and it will lead to style issues.

Reply:
Hi @sergei-deriv I imported the scss file inside the component which is using it if you check, but when I was checking other file's structure I saw they imported the scss in the index as well so I followed the same to not break anything. So I will be removing it from here if that's the case , thank you @sergei-deriv